paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -322,7 +324,7 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
                 bankName = result.bankName,
                 last4 = result.last4,
                 intentId = result.intent.id,
-                primaryButtonText = buildPrimaryButtonText(),
+                primaryButtonText = buildPrimaryButtonText().resolve(application),
```

Comment:
Can we propagate these?

Reply:
How do you mean?

Reply:
Oh right, that's what @samer-stripe mentioned in his comment that he'll need to update because `USBankAccountFormScreenState.MandateCollection` is `Parcelable` but `ResolvableString` is not

Reply:
Have a draft PR to make this possible: https://github.com/stripe/stripe-android/pull/8715

Reply:
Updated 👍